src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -78,28 +85,80 @@ class Widget_List extends Widget_Abstract {
 		'jsonld_enable'        => 'tribe_is_truthy',
 	];
 
-	/**
-	 * @todo update in TEC-3612 & TEC-3613
-	 *
-	 * {@inheritDoc}
-	 */
-	public function setup() {
-
-	}
-
 	/**
 	 * {@inheritDoc}
 	 */
-	public function get_arguments() {
+	public function get_arguments( $instance = [] ) {
 		$arguments = $this->arguments;
 
 		$arguments['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
 		// @todo update name once this widget is ready to replace the existing list widget.
 		$arguments['name']                          = esc_html__( 'Events List V2', 'the-events-calendar' );
 		$arguments['widget_options']['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
 
+		// Setup default title.
+		$arguments['title']        = __( 'Upcoming Events', 'the-events-calendar' );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 8 spaces

<sub>hash: d0a3c0cddfa047c19b0db57298614380</sub>